ucm2/MediaTek/mt8390-evk/mt8390-evk.conf

```diff
@@ -0,0 +1,40 @@
+Syntax 4
+
+SectionUseCase."HiFi" {
+	File "/MediaTek/mt8390-evk/HiFi.conf"
+	Comment "Play high quality music"
+}
+
+BootSequence [
+	cset "name='Headset Volume' 2"
+	cset "name='Lineout Volume' 10"
```

Comment:
Why the hardware volume controls are not connected to devices ?

Reply:
Agreed, I will move the hardware volume controls to the EnableSequence of the corresponding device.

Reply:
Done

Reply:
The control initialization should be in BootSequence, but then use `PlaybackMixerElem` or `CaptureMixerElem` values (strings) to pass this information to the sound server. In this way, the sound server can use this control for the hardware volume control.

Reply:
Hi @perexg,

I only use a fixed value in the configuration because the register value of the volume control on the SoC is opposite to the mixer value from loud to quiet.

If the hardware volume controls are connected to devices, there will be a conflict when controlling the volume through the sound server.

Reply:
> I only use a fixed value in the configuration because the register value of the volume control on the SoC is opposite to the mixer value from loud to quiet.

It seems dangerous. The driver should correct this (low value = quiet).